File: app/src/main/java/crypto/delta/exchange/openexchange/pojo/position/ConstituentExchange.java

package crypto.delta.exchange.openexchange.pojo.position;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ConstituentExchange {

    @SerializedName("weight")
    @Expose
    private Integer weight;
    @SerializedName("exchange")
    @Expose
    private String exchange;
    @SerializedName("health_interval")
    @Expose
    private Integer healthInterval;

    public Integer getWeight() {
        return weight;
    }

    public void setWeight(Integer weight) {
        this.weight = weight;
    }

    public String getExchange() {
        return exchange;
    }

    public void setExchange(String exchange) {
        this.exchange = exchange;
    }

    public Integer getHealthInterval() {
        return healthInterval;
    }

    public void setHealthInterval(Integer healthInterval) {
        this.healthInterval = healthInterval;
    }

}

File: app/src/main/java/crypto/delta/exchange/openexchange/pojo/products/ConstituentExchange.kt
package crypto.delta.exchange.openexchange.pojo.products

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ConstituentExchange {
    @SerializedName("weight")
    @Expose
    var weight: Double? = null

    @SerializedName("exchange")
    @Expose
    var exchange: String? = null

    @SerializedName("health_interval")
    @Expose
    var healthInterval: Int? = null

    @SerializedName("health_priority")
    @Expose
    var healthPriority: Int? = null

    @SerializedName("toSym")
    @Expose
    var toSym: String? = null

    @SerializedName("fromSym")
    @Expose
    var fromSym: String? = null

}